netpyne/batch.py

```diff
@@ -96,13 +96,30 @@ def run(self):
             # iterate over all param combinations
             if self.method == 'grid':
                 groupedParams = False
+<<<<<<< HEAD
```

Comment:
this line seems like was added after merging? think needs to be removed right? did you try running it after the merge to make sure still worked?